presto-docs/src/main/sphinx/release/release-345.md

```diff
@@ -0,0 +1,47 @@
+# Release 345 (XX Oct 2020)
+
+## General Changes
+
+* Add {func}`concat_ws` function. ({issue}`4680`)
+* Add support for {func}`extract` for `time with time zone` values with precision other than 3. ({issue}`5539`)
+* Add dynamic filtering support for right joins. ({issue}`5555`) 
+* Add support for catalog session property to file based session property manager. ({issue}`5330`)
+* Add support for catalog session property to database based session property manager. ({issue}`5330`)
+* Improve query scalability by increasing broadcast join buffer size to 200MB.
+  Broadcast join buffer size can be set using `sink.max-broadcast-buffer-size` feature config. ({issue}`5551`)
+* Improve query performance by allowing larger dynamic filters. ({issue}`5529`)
+* Improve dynamic filtering across connectors by adding implicit cast support for dynamic filters. ({issue}`5461`)
```

Comment:
How does it sound from end user perspective?

maybe something about: improve performance for join queries where join keys are not of the same type (eg integer and bigint)